src/kirby/components/badge/badge.component.tns.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../scss/native-imports";
+
+.kirby-badge Label { 
+  font-size: font-size("xxs");
+  border-radius: size("xs");
```

Comment:
Consider using a really big number (eg. 500) instead, to make sure borders are rounded when height changes (e.g. for wrapping text)

Reply:
As long as you use a unit other than % then the borders will always be rounded by the amount specified no matter the size of the element.